### ISSUE TITLE
Allow setting of jail parameters automatically

### ISSUE THREAD

**Issue by tobiastom** [BastilleBSD/bastille#259]
**Is your feature request related to a problem? Please describe.**
I'm trying to create a postgresql jail. As it needs special jail parameters (`allow.sysvipc=1`), I have to manually modify them every time the jail is restarted.

**Describe the solution you'd like**
Allow the `Bastillefile` to specify these parameters and load them automatically when the jail starts.

**Additional context**
Right now I'm using something like this to make it easier for me:

```bash
for jid in $(jls | grep postgresql | awk '{print $1}'); do
    jail -m jid=$jid allow.sysvipc=1
done
```

**Comment by xaque208** [comment]
Neat idea.  Is this a matter of writing the correct jail.conf?

**Comment by tobiastom** [comment]
Yes, indeed. What I'd like to use (inside a `Bastillefile`) is something like this:

```bash
CONFIG sysvshm=new
CONFIG sysvsem=new
```

This should then write this to the `jail.conf` and use it on the restart of the jail.

**Comment by chriswells0** [comment]
I should just read all your feature requests so I'll know what I need. ;)

I didn't know about this request until just now, but I submitted the feature this morning as #283. There is also #288 to fix an issue in that merge.

From a terminal:
```
bastille config TARGET set allow.sysvipc 1
```

In a Bastillefile:
```
CONFIG set allow.sysvipc 1
```

**Comment by tobiastom** [comment]
@chriswells0 as always, works excellent for me. Thanks for implementing this. One step closer to finish my jail automation vision. :)

Still, as always, one more thing: what do you think is the best practice for the `Bastillefile`? I'm trying to create a PostgreSQL jail. My file looks a little bit like this:

```
OVERLAY usr
OVERLAY root

PKG postgresql13-server postgresql13-contrib

SYSRC postgresql_enable=YES

CONFIG set sysvshm new
CONFIG set sysvsem new

SERVICE postgresql initdb

# … more to come.
```

The problem is now that the config is changed successfully, but `initdb` does not work as the jail is not yet restarted.
As we can set multiple values, I can see that it might not be the best idea to restart the jail automatically after setting a value. Still, without a restart the bastille file is kind of broken. 

What about a `RESTART` command inside the `Bastillefile`? 🤪

**Comment by cedwards** [comment]
I agree. Let's support `START`, `STOP` and `RESTART` to mirror the sub-commands.

**Comment by chriswells0** [comment]
The `Bastillefile` is processed expecting the 1st word on each line to be a Bastille subcommand, so `START`, `STOP`, and `RESTART` already work inside a `Bastillefile`. ;) Every subcommand that takes a target as its first argument should work in a `Bastillefile`. Any that don't should be treated as bugs.

One thing I would suggest is to use `CP` or `COPY` instead of `OVERLAY`. That keeps you in the mindset of using the subcommands, which I believe is easier when switching back and forth (CLI vs. Bastillefile).

**Comment by cedwards** [comment]
Per some out-of-band discussion we want to support all subcommands in Bastillefiles. This includes bootstrap, create, start, stop, destroy, etc. That support requires some separate changes.

We will standardize on subcommands (cp > overlay, etc).

In terms of supporting Bastille `config` I'm considering this resolved; closing.
fixed in #283 / #288 .

**Comment by tobiastom** [comment]
Thank you for your work on this feature.
